crates/project_panel/src/project_panel.rs

```diff
@@ -1249,6 +1236,27 @@ impl ProjectPanel {
             cx.notify();
         }
     }
+    fn create_relative_path(
+        &self,
+        source_entry: &SelectedEntry,
+        (worktree, target_entry): (Model<Worktree>, &Entry),
+        cx: &AppContext,
+    ) -> Option<PathBuf> {
+        let source_path = worktree.read(cx).absolutize(&target_entry.path).ok()?;
```

Comment:
> `source_path`

It is not related to source though, but to the target?
Let's name it a bit better.

Reply:
done